docs/releases/5.0.txt

```diff
@@ -192,6 +192,10 @@ Minor features
 
 * Added support for GEOS 3.12.
 
+* The new
+  :meth:`.GEOSGeometry.equals_identical()` method returns ``True`` if the two
```

Comment:
`()` are unnecessary, sphinx adds them automatically to `:meth:` and `:func:` directives:
```suggestion
  :meth:`.GEOSGeometry.equals_identical` method returns ``True`` if the two
```